iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -2030,6 +2034,8 @@
 		3454D7981E07F045004AF2AD /* Categories */ = {
 			isa = PBXGroup;
 			children = (
+				ED8AC9092ACD9DA100EECC90 /* UIStackView+ArrangedSubviewWithSeparator.swift */,
+				ED8AC90C2ACD9DA100EECC90 /* UIView+AddLine.swift */,
```

Comment:
Should these additions be a part of another commit?

Reply:
My mistake. I will fix that!